README.md

```diff
@@ -1 +1,39 @@
-# Cinema app (in progress)
+![GitHub Logo](cinema.jpg)
+
+# Cinema app
+
+This application emulates the workflow of cinema and allows us to manage a database of users that have the possibility to choose movie sessions and place orders with the help of http requests from the client.
+The users have two distinct roles with limited access to url endpoints.
+The application is based on SOLID principles and N-tier architecture with 3 levels such as DAO layer with CRUD operations, Service layer, Controller layer. 
+
+# Technologies used:
+
+* Java 11
+* Hibernate
+* Spring Core
+* Spring MVC
+* Spring Web
+* Spring Security
+* MySQL 
+* Maven Checkstyle Plugin
+* Apache Tomcat
+
+# Features implemented:
+
+* DTO Validation
+* Password hashing
+* Access distribution among USER and ADMIN roles
```

Comment:
we can also add detailed info about which functionality is available to USER, ADMIN, both roles